posthog/api/capture.py

```diff
@@ -187,6 +193,18 @@ def _get_sent_at(data, request) -> Tuple[Optional[datetime], Any]:
         )
 
 
+def _check_token_shape(token: str) -> Optional[str]:
+    if not token:
+        return "empty"
+    if token.startswith("phx_"):  # Used by previous versions of the zapier integration, should not happen now
+        return "personal_token"
+    if len(token) > 64:
+        return "too_long"
```

Comment:
This one should probably be first? I assume there could be a very long token that starts with `phx`.

Reply:
Agreed, personal tokens are also < 64 chars (same logic as new project tokens)